apps/testingaccessibility/README.md

```diff
@@ -12,34 +18,32 @@ bin/validate
 
 Missing system dependencies should be installed. They will be assumed below.
 
-## Install Dependencies, Build, and Test
+## Serverless Access
 
-From the root of the project install all of the dependencies for the entire monorepo by running the following command:
+You may need access to the following serverless accounts to run the app:
+- Stripe (for Stripe env vars and Stripe CLI)
+- Vercel (for env vars, e.g. `POSTMARK_KEY`)
+- Planetscale (if needed) or use MySQL in Docker
```

Comment:
I find it helpful to have a listing of these kinds of external dependencies upfront.